### ISSUE TITLE
Make ForceField look automatically in data/forcefield directory instead of data/

### ISSUE THREAD

**Issue by andrrizzi** [openforcefield/openforcefield#163]
Currently when we need to prepend `forcefield/` to all offxml files defined in the repo
```
forcefield = ForceField('forcefield/Frosst_AlkEthOH_parmAtFrosst.offxml')
```
Should we make `ForceField` look automatically in `data/forcefield/` in [this snippet](https://github.com/openforcefield/openforcefield/blob/77e66993daacdfba61d89888cca82223452b621b/openforcefield/typing/engines/smirnoff/forcefield.py#L556) instead of just `data/` and make the call a little simpler?
```
forcefield = ForceField('Frosst_AlkEthOH_parmAtFrosst.offxml')
```
Or is there a reason I don't see to avoid this?

**Comment by andrrizzi** [comment]
I'm going to implement this in the next couple of days if there are no objections to this simplification.

**Comment by davidlmobley** [comment]
I think this is a good idea.

@jchodera  also had the suggestion that eventually we make it so it would be possible to register other force fields to the search path easily, as eventually we will have many force field releases which each can be separately installed and you'd like to have a way to register the path to search for those, perhaps even automatically when they are installed. If you know an easy way to do this, it might be worth thinking about the same time. He seemed to know how to do it.

**Comment by andrrizzi** [comment]
Ok, thanks! Yes, I see @jchodera captured the idea in this issue #117. Probably low priority right now, but good to keep on the radar.

**Comment by andrrizzi** [comment]
This was implemented in #163 .
